### PR TITLE
[SDK-1980] Add support for Session Deadline

### PR DESCRIPTION
**Pull request by MichalMilewiczYoti**
Add support for Session Deadline

### CODE REVIEW

File: yoti_python_sdk/doc_scan/session/create/session_spec.py [hunk 1]
```diff
@@ -196,6 +212,18 @@ def with_client_session_token_ttl(self, value):
         self.__client_session_token_ttl = value
         return self
 
+    def with_session_deadline(self, value):
+        """
+        Sets the session deadline
```
Comment (echarrod) [inline yoti_python_sdk/doc_scan/session/create/session_spec.py:217]:
```suggestion
        Sets the deadline that the session needs to be completed by. Can be used as an alternative to with_client_session_token_ttl
```

File: yoti_python_sdk/doc_scan/session/retrieve/create_session_result.py [hunk 2]
```diff
@@ -49,3 +50,13 @@ def session_id(self):
         :rtype: str or None
         """
         return self.__session_id
+
+    @property
+    def session_deadline(self):
+        """
+        Session deadline created session
```
Comment (echarrod) [inline yoti_python_sdk/doc_scan/session/retrieve/create_session_result.py:57]:
```suggestion
        The deadline that the session needs to be completed by
```

**Review by echarrod — APPROVED** [review]
Just a couple of points it would be good to expand the docs a bit more, looks good otherwise

**Review by ShamariYoti — APPROVED** [review]

File: yoti_python_sdk/doc_scan/session/create/session_spec.py (hunk 1 above)
Comment (MichalMilewiczYoti) [reply yoti_python_sdk/doc_scan/session/create/session_spec.py:217]: done

File: yoti_python_sdk/doc_scan/session/retrieve/create_session_result.py (hunk 2 above)
Comment (MichalMilewiczYoti) [reply yoti_python_sdk/doc_scan/session/retrieve/create_session_result.py:57]: done

**Comment by echarrod** [comment]
Think we're OK to merge this one now @MichalMilewiczYoti, LGTM

**Pull request merged**